moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -586,15 +595,22 @@ const ProjectFundingTable = () => {
             />
             <Box component={"ul"} className={classes.chipContainer}>
               <Typography className={classes.fieldLabel}>Task order</Typography>
-              {taskOrderData.map((task) => (
-                <li key={task.id}>
-                  <Chip
-                    label={task.display_name}
-                    onDelete={() => handleTaskOrderDelete(task)}
-                    className={classes.chip}
-                  />
-                </li>
-              ))}
+              <DeleteConfirmationModal
+                type="tag"
+                submitDelete={() => handleTaskOrderDelete(deleteConfirmationId)}
+                isDeleteConfirmationOpen={isDeleteConfirmationOpen}
+                setIsDeleteConfirmationOpen={setIsDeleteConfirmationOpen}
+              >
+                {taskOrderData.map((task) => (
+                  <li key={task.id}>
```

Comment:
this issue pre-dates your work, but i'm seeing a unique key prop error here. the problem is that `task.id` is `undefined` for all of these records. you can switch this to `task.task_order` to clear i tup.